src/LocalRegistry.jl

```diff
@@ -195,6 +198,14 @@ function do_register(package, registry;
         package_repo = repo
     end
 
+    # Check that `package_repo` is not a relative path. Relative paths do not work
+    # currently with Pkg (Pkg issue #677).
+    # This approach here does not always work, because the path may be relative to another
+    # location. Further, nearly all strings are valid paths in Unix, including URLs.
```

Comment:
Yeah... maybe I should remove this check altogether. I wish it was possible to simply check if a string was a local path, but that is not possible because Julia does not provide a path type to disambiguate this.
The idea is that documentation is good, because it enables people to not screw up but throwing errors is better, because it forces them not to.